src/notation/view/abstractnotationpaintview.cpp

```diff
@@ -721,11 +721,23 @@ RectF AbstractNotationPaintView::scrollableAreaRect() const
 {
     TRACEFUNC;
     RectF viewport = this->viewport();
+    RectF contentRect = notationContentRect();
     qreal overscrollFactor = configuration()->isLimitCanvasScrollArea() ? 0.0 : SCROLL_LIMIT_OFF_OVERSCROLL_FACTOR;
 
     qreal overscrollX = viewport.width() * overscrollFactor;
     qreal overscrollY = viewport.height() * overscrollFactor;
 
+    // contentRect is Vertically or Horizontally larger than viewport
+    // 1) Continuous Horizontal View
+    if (notation()->viewMode() == engraving::LayoutMode::LINE && viewport.width() < contentRect.width()) {
+        return notationContentRect().adjusted(0, -overscrollY, 0, overscrollY);
```

Comment:
This solution has two potential disadvantages:
- if `viewport.width() < contentRect.width()` _and_ `viewport.height() < contentRect.height()`, you would expect that the non-adjusted `notationContentRect()` is returned, but instead you get `notationContentRect().adjusted(0, -overscrollY, 0, overscrollY);`
- if `viewport.width()` is just _slightly_ larger than `contentRect.width()`, then it will immediately be possible to scroll all the way past the end of the notation. 

Would it be an idea to change only the line where `overscrollFactor` is defined, so that it is `SCROLL_LIMIT_OFF_OVERSCROLL_FACTOR` when the scroll area is not limited and the view mode is not continuous view, and `0.0` otherwise?

Reply:
Change in overscrollFactor conflicts with possible zoom out percentage. To overcome removing zoom out percentages, I implemented user can scroll horizontally or vertically to page border only if page fits into viewbox. If page is smaller than view box than he can scroll outside as well. 
I will look into your idea once again